### ISSUE TITLE
Links without event action

### ISSUE THREAD

**Issue by dzhaotbs** [wet-boew/web-reporting#48]
Home>Treasury Board>Feb,2013>Manage>Manage Websites

After login as admin  Home and the report period (Feb,2013) have link anchors but these links have no event with them. These anchors should be removed if there no events with them.
